webui/react/src/components/TaskCard.tsx

```diff
@@ -15,12 +15,13 @@ import css from './TaskCard.module.scss';
 const TaskCard: React.FC<RecentTask> = (props: RecentTask) => {
   let [ hasProgress, isComplete ] = [ false, false ];
   if (isExperimentTask(props)) {
-    hasProgress = props.progress != null;
+    hasProgress = !!props.progress;
```

Comment:
non-blocking: This seems like a TS anti-pattern unless I'm misunderstanding. Not blocking for this ticket.

Reply:
On the new api, the `progress` field seems to always return `0` (previously if there is no progress it would come through as `null` or `undefined`). This caused task cards to show `0%` all the time regardless of state.

Reply:
Wait, does this mean it doesn't report progress at all?

Reply:
It only reports progress if it's greater than 0